### PR TITLE
fix that compiles failed on macOS

### PR DESCRIPTION
**Pull request by LucasLi1024**

### CODE REVIEW

File: cmake/ProjectSDF.cmake [hunk 1]
```diff
@@ -4,9 +4,9 @@ include(GNUInstallDirs)
 if("${CMAKE_HOST_SYSTEM_NAME}" MATCHES "Linux")
     set(SDF_LIB_NAME libsdf-crypto_arm.a)
 elseif(APPLE)
-    message(FATAL "HSM  SDF only support Linux, the ${CMAKE_HOST_SYSTEM_NAME} ${ARCHITECTURE} is not supported.")
+    message(FATAL " HSM  SDF only support Linux, the ${CMAKE_HOST_SYSTEM_NAME} ${ARCHITECTURE} is not supported.")
```
Comment (bxq2011hust) [inline cmake/ProjectSDF.cmake:7]:
```suggestion

```

File: cmake/ProjectSDF.cmake [hunk 2]
```diff
@@ -4,9 +4,9 @@ include(GNUInstallDirs)
...
 else()
-    message(FATAL "HSM  SDF only support Linux, the ${CMAKE_HOST_SYSTEM_NAME} ${ARCHITECTURE} is not supported.")
+    message(FATAL " HSM  SDF only support Linux, the ${CMAKE_HOST_SYSTEM_NAME} ${ARCHITECTURE} is not supported.")
```
Comment (bxq2011hust) [inline cmake/ProjectSDF.cmake:9]:
```suggestion

```

Comment (LucasLi1024) [reply cmake/ProjectSDF.cmake]: fixed

File: cmake/ProjectSDF.cmake (hunk 1 above)
Comment (LucasLi1024) [reply cmake/ProjectSDF.cmake]: fixed

**Review by bxq2011hust — APPROVED** [review]

**Review by kyonRay — APPROVED** [review]

**Pull request merged**